src/Commands/CleanCategory.php

```diff
@@ -21,92 +21,219 @@
 use Category;
 use Configuration;
 use FOP\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
 final class CleanCategory extends Command
 {
+    /**
+     * @var array possible command
+     */
+    const ALLOWED_COMMAND = ['status', 'toggle', 'enable-noempty', 'disable-empty'];
+
     /**
      * {@inheritdoc}
      */
     protected function configure()
     {
-        $this->setName('fop:clean-category')
-            ->setDescription('Active or desactive categorys depending on products')
-            ->setHelp('This command desactive final categories witout product, and active final catgories with product')
+        $this->setName('fop:category')
+            ->setDescription('Manage your categories')
+            ->setHelp('This command active or desacative catégory and desactive final categories witout product or active final catgories with product')
```

Comment:
Juste quelques fautes de frappes ici :smile: 
This command active or desactive category and desactive final categories without product or active final categories with product